src/main/kotlin/org/castlebet/chess/Application.kt

```diff
@@ -35,16 +38,6 @@ fun Application.main() {
     install(ContentNegotiation) {
         json(Json)
     }
-    install(CORS) {
```

Comment:
No need for CORS in this webapp, we don't have to share resources with other domains